### ISSUE TITLE
Obj-C Interface Header missing API methods

### ISSUE THREAD

**Issue by davidgovea** [electrode-io/electrode-native#1640]
Hi electrode team!
I am not sure if this is an actual bug, or just something with my setup -- I will be generating some test projects from scratch to help isolate the issue.

After upgrading from 0.37 to 0.41, my Objective-C Interface Header (`ElectrodeContainer-Swift.h`) is missing API methods.

Here is an example showing the diff, for an API that has 3 methods: `dismiss`, `notifyClose`, and `remove`.

**`ElectrodeContainer-Swift.h`** of built ElectrodeContainer.framework, compared with previous built version
![Screen Shot 2020-05-26 at 10 51 56 AM](https://user-images.githubusercontent.com/489896/82934212-fe835f80-9f3f-11ea-96c4-3dd2d63ee0b2.png)

The `registerXXXRequestHandler` and `unregisterXXXRequestHandler` methods are still there, but the "actual" API call hooks are missing.

I have regenerated the API recently, and now looking at it, it appears that the signature changed slightly. In Swift, the underscore means that the parameter has no "external name", right? Maybe this has affected the interface header?

**`XXXApi.swift`**, after recent regeneration
![Screen Shot 2020-05-26 at 11 06 14 AM](https://user-images.githubusercontent.com/489896/82934991-3fc83f00-9f41-11ea-9260-5aaeb560fd31.png)

I am going to try a few things:
* Using older API files
* Regenerating fresh container / example API

Hopefully this is helpful & not just an error on my end. I'd guess that if consuming apps are using swift only, this issue may not have been noticed (I need to get to full swift!).

Thanks for reading! Really appreciate the work of @belemaire & @friederbluemle, and the continued OSS investment by @walmartlabs in general.

**Comment by friederbluemle** [comment]
Thanks for the report and investigation @davidgovea :+1: It does look like something has changed in the platform. We'll get back to you by end of day!

**Comment by friederbluemle** [comment]
@davidgovea - I created an api locally, but was not able to reproduce the problem yet. Will continue tomorrow.

Regarding the underscores in functions: It does look like SwiftLint (applied in 7270c5dbf4bed4861a864a670cb315f049455c06) replaced the _parameter names_ with underscores in methods that did not use the parameter in the **implementation** ([example](https://github.com/electrode-io/electrode-native/pull/1420/commits/7270c5dbf4bed4861a864a670cb315f049455c06#diff-96e879ea810b99404db586014910887f)). However, the **external** _argument labels_ are not modified.

Swift Docs:

https://docs.swift.org/swift-book/LanguageGuide/Functions.html#//apple_ref/doc/uid/TP40014097-CH10-ID166

**Comment by davidgovea** [comment]
Thank you for investigating! This was an issue on my end - sorry!

There was something wrong with the way I updated my API libraries. I did a manual bump of the react-native-electrode-bridge package, and did an `ern regen-api`. It's also possible that I had a mismatched version of the ern platform installed, since I've been switching between prod releases and dev 1000.0.0 a lot.

A freshly-generated miniapp (with the default EnNavigationAPI) worked fine, and I was also able to just use my old API packages with the latest ern platform.

It looks like the freshly-generated APIs do have a slightly different naming convention in the generated Objective-C header. The change in the swift naming has resulted in functions without the `WithCompletionHandler` suffix in ObjC. It's nicer that way, tbh, and I was able to maintain my old function names for backward-compat by simply not regenerating the API handles.
